packages/form-config/src/components/Antd/ReleaseList/utils.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Dictionary } from '@onaio/utils';
+import { ManifestReleasesTypes } from '../../../ducks/manifestReleases';
+import { TableActions } from './TableActions';
+import { formatDate } from '../../../helpers/utils';
+
+/**
+ * Return table columns
+ *
+ * @param {string} viewReleaseURL URL to view the release details
+ * @param {Dictionary} sortedInfo object containing sorting order information
+ * @returns {Dictionary[]} table columns
+ */
+export const getTableColumns = (viewReleaseURL: string, sortedInfo?: Dictionary): Dictionary[] => {
```

Comment:
does antd provide a better type that we can use for the return type here

Reply:
@p-netm I haven't found a better type for this one